diff --git a/omega/template.php b/omega/template.php
index a4ce176..7bb73f2 100644
--- a/omega/template.php
+++ b/omega/template.php
@@ -27,19 +27,18 @@ function omega_alpha_zones_alter(&$zones, $theme) {
  */
 function omega_alpha_preprocess_html(&$vars) {
   $theme = alpha_get_theme();
+  $vars['doctype'] = '<!DOCTYPE html>' . "\n";
   $vars['rdf'] = new stdClass;
-
-  if (module_exists('rdf')) {
-    $vars['doctype'] = '<!DOCTYPE html PUBLIC "-//W3C//DTD HTML+RDFa 1.1//EN">' . "\n";
-    $vars['rdf']->version = ' version="HTML+RDFa 1.1"';
-    $vars['rdf']->namespaces = $vars['rdf_namespaces'];
-    $vars['rdf']->profile = ' profile="' . $vars['grddl_profile'] . '"';
-  }
-  else {
-    $vars['doctype'] = '<!DOCTYPE html>' . "\n";
-    $vars['rdf']->version = '';
-    $vars['rdf']->namespaces = '';
-    $vars['rdf']->profile = '';
+  $vars['rdf']->version = '';
+  $vars['rdf']->namespaces = '';
+  $vars['rdf']->profile = '';
+
+  // Serialize RDF Namespaces into an RDFa 1.1 prefix attribute.
+  if (function_exists('rdf_get_namespaces')) {
+    foreach (rdf_get_namespaces() as $prefix => $uri) {
+      $prefixes[] = $prefix . ': ' . $uri;
+    }
+    $vars['rdf']->namespaces = ' prefix="' . implode(' ', $prefixes) . '"';
   }
 
   if (alpha_library_active('omega_mediaqueries')) {
