diff --git a/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.features.field.inc b/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.features.field.inc
old mode 100755
new mode 100644
index 23d6ab8..9108439
--- a/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.features.field.inc
+++ b/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.features.field.inc
@@ -61,7 +61,7 @@ function cod_bof_field_default_fields() {
           'module' => 'text',
           'settings' => array(),
           'type' => 'text_default',
-          'weight' => 0,
+          'weight' => '0',
         ),
         'listing' => array(
           'label' => 'above',
@@ -105,7 +105,7 @@ function cod_bof_field_default_fields() {
   $fields['node-bof_session-field_session_timeslot'] = array(
     'field_config' => array(
       'active' => '1',
-      'cardinality' => '-1',
+      'cardinality' => '1',
       'deleted' => '0',
       'entity_types' => array(),
       'field_name' => 'field_session_timeslot',
@@ -138,7 +138,7 @@ function cod_bof_field_default_fields() {
           'view' => array(
             'args' => array(),
             'display_name' => 'entityreference_1',
-            'view_name' => 'cod_session_timeslot',
+            'view_name' => 'cod_bof_timeslot',
           ),
         ),
         'target_type' => 'field_collection_item',
@@ -149,7 +149,7 @@ function cod_bof_field_default_fields() {
     'field_instance' => array(
       'bundle' => 'bof_session',
       'default_value' => NULL,
-      'default_value_function' => '',
+      'default_value_function' => 'entityreference_prepopulate_field_default_value',
       'deleted' => '0',
       'description' => '',
       'display' => array(
@@ -157,10 +157,11 @@ function cod_bof_field_default_fields() {
           'label' => 'above',
           'module' => 'entityreference',
           'settings' => array(
-            'link' => FALSE,
+            'links' => 0,
+            'view_mode' => 'full',
           ),
-          'type' => 'entityreference_label',
-          'weight' => 3,
+          'type' => 'entityreference_entity_view',
+          'weight' => '3',
         ),
         'listing' => array(
           'label' => 'above',
@@ -182,7 +183,10 @@ function cod_bof_field_default_fields() {
       'settings' => array(
         'behaviors' => array(
           'prepopulate' => array(
-            'status' => 0,
+            'action' => 'disable',
+            'fallback' => 'none',
+            'skip_perm' => '0',
+            'status' => 1,
           ),
         ),
         'user_register_form' => FALSE,
diff --git a/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.features.inc b/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.features.inc
old mode 100755
new mode 100644
diff --git a/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.info b/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.info
old mode 100755
new mode 100644
index 866fa82..a1c6a55
--- a/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.info
+++ b/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.info
@@ -3,15 +3,17 @@ description = BoFs are self-scheduled, unconference style sessions
 core = 7.x
 package = COD
 php = 5.2.4
+version = 7.x-1.0-alpha1
+project = cod_bof
 dependencies[] = cod_session
-dependencies[] = commons_wysiwyg
 dependencies[] = entityreference
+dependencies[] = entityreference_prepopulate
 dependencies[] = features
 dependencies[] = field_group
 dependencies[] = options
 dependencies[] = strongarm
 dependencies[] = views
-dependencies[] = views_field_view
+datestamp = 1352511671
 features[ctools][] = field_group:field_group:1
 features[ctools][] = strongarm:strongarm:1
 features[ctools][] = views:views_default:3.0
@@ -40,11 +42,3 @@ features[variable][] = node_options_bof_session
 features[variable][] = node_preview_bof_session
 features[variable][] = node_submitted_bof_session
 features[views_view][] = cod_bof_schedule
-
-
-; Information added by drush on 2012-11-10
-version = "7.x-1.0-alpha1+0-dev"
-core = "7.x"
-project = "cod_support"
-datestamp = "1352511671"
-
diff --git a/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.module b/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.module
old mode 100755
new mode 100644
diff --git a/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.strongarm.inc b/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.strongarm.inc
old mode 100755
new mode 100644
index 2f3c87d..ca3972a
--- a/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.strongarm.inc
+++ b/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.strongarm.inc
@@ -71,7 +71,29 @@ function cod_bof_strongarm() {
   $strongarm->api_version = 1;
   $strongarm->name = 'field_bundle_settings_node__bof_session';
   $strongarm->value = array(
-    'view_modes' => array(),
+    'view_modes' => array(
+      'teaser' => array(
+        'custom_settings' => TRUE,
+      ),
+      'listing' => array(
+        'custom_settings' => TRUE,
+      ),
+      'full' => array(
+        'custom_settings' => FALSE,
+      ),
+      'rss' => array(
+        'custom_settings' => FALSE,
+      ),
+      'search_index' => array(
+        'custom_settings' => FALSE,
+      ),
+      'search_result' => array(
+        'custom_settings' => FALSE,
+      ),
+      'token' => array(
+        'custom_settings' => FALSE,
+      ),
+    ),
     'extra_fields' => array(
       'form' => array(
         'title' => array(
@@ -108,7 +130,7 @@ function cod_bof_strongarm() {
   $strongarm->name = 'node_options_bof_session';
   $strongarm->value = array(
     0 => 'status',
-    1 => 'promote',
+    1 => 'revision',
   );
   $export['node_options_bof_session'] = $strongarm;
 
diff --git a/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.views_default.inc b/docroot/profiles/cod/modules/contrib/cod_support/cod_bof/cod_bof.views_default.inc
old mode 100755
new mode 100644
